helpers/hllm.py

```diff
@@ -273,92 +273,121 @@ def _call_api_sync(
 # Cost tracking
 # #############################################################################
 
-# TODO(*): Convert this into a class to track costs?
+_CURRENT_OPENAI_COST = None
 
 
-_CURRENT_OPENAI_COST = None
+# #############################################################################
+# LLMCostTracker
+# #############################################################################
 
 
-def start_logging_costs() -> None:
-    global _CURRENT_OPENAI_COST
-    _CURRENT_OPENAI_COST = 0.0
+class LLMCostTracker:
```

Comment:
The goal is to make this class the "state", so this guy should have the state in the init.

Then users create one for all the time that they need to track by passing this to get_completion

Does the design make sense?